tests/integration/injection-test.js

```diff
@@ -66,13 +66,23 @@ class ChromeApi {
       },
     };
 
+    const contextMenuListeners = [];
+
     this.contextMenus = {
       remove(name) {
         delete self.registeredContextMenus[name];
       },
 
       create(menu) {
         self.registeredContextMenus[menu.id] = menu;
+        self.registeredContextMenus[menu.id].onclick = () =>
+          contextMenuListeners.forEach((fn) => fn({ menuItemId: menu.id }));
+      },
+
+      onClicked: {
```

Comment:
I assume you tested this to ensure it still found the component in the tree and selected it?

Reply:
i tested it now :)
works